.env.template

```
@@ -1,115 +1,124 @@
+LLM_MAX_TOKENS=4096
+LLM_TIMEOUT=60
+
+# Optional OpenAI-compatible overrides
+LLM_API_BASE=
+LLM_API_VERSION=
+
+# Capability overrides (true / false)
+LLM_SUPPORTS_PYDANTIC=
+LLM_SUPPORTS_STREAMING=true
+LLM_SUPPORTS_VISION=
+LLM_SUPPORTS_TOOL_PARALLELISM=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LLM_SUPPORTS_TOOL_PARALLELISM key should go before the LLM_SUPPORTS_VISION key

(UnorderedKey)